Makefile

```diff
@@ -45,6 +48,11 @@ GO_BUILD := $(GO) build -ldflags="-s -w -X $(PACKAGE)/pkg/version.Version=$(VERS
 .PHONY: all
 all: binaries manpages
 
+.PHONY: help
+help:
+	@echo  '  binaries        - Build all binaries'
+	@echo  '  manpages        - Build manual pages'
+
 exe: _output/bin/limactl$(exe)
 
 .PHONY: minimal
```

Comment:
`make minimal` should now generate a new `.config` and run `make binary` ?

Reply:
Not everything is configurable, and there is no mechanism for "host arch"